test/unit/detect-transaction-example-numbers-test.coffee

```diff
@@ -17,47 +17,21 @@ scenario = (description, {actionContent, examples, exampleNumbersPerTransaction}
       ### Catch a Smurf [POST]
       #{actionContent}
     """
-    action = undefined # API Blueprint AST
-    transition = undefined # API Elements (a.k.a. Refract)
+
+    transitionElements = undefined
+    transactionExampleNumbers = undefined
 
     beforeEach((done) ->
-      options = {type: 'ast', generateSourceMap: true}
-      drafter.parse(apiBlueprint, options, (err, parseResult) ->
-        return done(err) if err
-        action = parseResult.ast.resourceGroups[0].resources[0].actions[0]
-        done()
-      )
-    )
-    beforeEach((done) ->
-      options = {type: 'refract', generateSourceMap: true}
-      drafter.parse(apiBlueprint, options, (err, parseResult) ->
-        return done(err) if err
-        transition = parseResult.content[0].content[0].content[0].content[0]
+      parse(apiBlueprint, (args...) ->
+        [error, {mediaType, apiElements}] = args
+        transitionElements = apiElements.findRecursive('transition').first()
```

Comment:
I'd suggest keeping `findRecursive` to a minimum until https://github.com/refractproject/minim/pull/137 is completed because `findRecursive` involves a lot of cloning otherwise.